alerts/src/commonMain/kotlin/Alerts.kt

```diff
@@ -189,13 +189,34 @@ abstract class BaseAlertBuilder {
      */
     fun addActions(actions: List<Alert.Action>) = apply { this.actions.addAll(actions) }
 
+    /**
+     * Builds an alert using DSL syntax
+     *
+     * @param initialize
+     * @return
+     */
+    fun alert(initialize: BaseAlertBuilder.() -> Unit): AlertInterface {
+        reset()
+        initialize()
```

Comment:
This pattern makes the builder non threadsafe.

That might be acceptable, but what will happen if we pop more than one dialog? Will the handlers also be reset? (seems like they are stored in actions).

Reply:
we should probably make a test for this behaviour, regardless of what happens.

Reply:
This is good point

Reply:
I think this might work now. The multithreaded testcase is hard, but the testcase for building two alerts in a row can still be added I think (unless I missed it somewhere).

Reply:
Definitely